SCClassLibrary/Common/Quarks/Git.sc

```diff
@@ -48,6 +48,22 @@ Git {
 		});
 		^nil
 	}
+	httpsString {
```

Comment:
I settled on `remoteAsHttpUrl` since `web` as a notion of the 90s IMHO does not tell anything about the used protocol. `git://` is as much web as `http://`.
for the records: technically, it should read `remoteAsHttpUri` but the class library has a lot of `url` mentioned where it should be `uri`, so I follow the masses here.